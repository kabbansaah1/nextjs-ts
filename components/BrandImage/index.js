import BrandImage from './BrandImage';

export default BrandImage;
