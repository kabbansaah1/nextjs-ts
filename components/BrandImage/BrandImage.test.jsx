import { render } from '@testing-library/react';
import BrandImage from './BrandImage';
import '@testing-library/jest-dom';

describe('Home', () => {
  it('renders BrandImage unchanged', () => {
    const { container } = render(<BrandImage />);

    expect(container).toMatchSnapshot();
  });
});
