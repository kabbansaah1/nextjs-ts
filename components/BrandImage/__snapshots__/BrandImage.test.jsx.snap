// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders BrandImage unchanged 1`] = `
<div>
  <div
    class="relative mb-16 flex place-items-center before:absolute before:h-[300px] before:w-[480px] before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[''] after:absolute after:-z-20 after:h-[180px] after:w-[240px] after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[''] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700/10 after:dark:from-sky-900 after:dark:via-[#0141ff]/40 before:lg:h-[360px]"
  >
    <img
      alt="Next.js Logo"
      class="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
      data-nimg="1"
      decoding="async"
      fetchpriority="high"
      height="37"
      src="/next.svg"
      style="color: transparent;"
      width="180"
    />
  </div>
</div>
`;
